src/firefly/js/ui/PositionFieldDef.js

```diff
@@ -68,6 +69,21 @@ export function formatTargetForHelp(wp) {
         '</div>';
 }
 
+function getEQJ2000(wp) {
+    const wpEQ = VisUtil.convert(wp, CoordinateSys.EQ_J2000);
+    const [ra, dec] = [wpEQ.getLon().toFixed(5), wpEQ.getLat().toFixed(5)];
+    const hmsRa = CoordUtil.convertLonToString(ra,  CoordinateSys.EQ_J2000);
+    const hmsDec = CoordUtil.convertLatToString(dec,  CoordinateSys.EQ_J2000);
+
+    const res = '<div  style=\"padding-top:6px;font-size:10px;\">' +
+        ra + ',&nbsp;' + dec + '&nbsp;Equ J2000' +
+        ' &nbsp;&nbsp;&nbsp;<i>or</i> &nbsp;&nbsp;&nbsp;' +
+        hmsRa + ',&nbsp;' + hmsDec + '&nbsp;&nbsp;Equ J2000' +
+        '</div>';
```

Comment:
This would probably be easier to read with template literals (backquote) strings.